languages/csharp/reference/exercise-concepts/nucleotide-count.md

```diff
@@ -0,0 +1,47 @@
+# Concepts of nucleotide-count
```

Comment:
Can you add a link to the v2 implementation right after the header? I'm going to do the same with my exercises as well.

Reply:
Done!